host-configs/TOTAL/pangea2-base.cmake

```diff
@@ -30,7 +30,7 @@ set(CUDA_ENABLED      "OFF"       CACHE PATH "" FORCE)
 set(CHAI_BUILD_TYPE   "cpu-no-rm" CACHE PATH "" FORCE)
 set(CHAI_ARGS         ""          CACHE PATH "" FORCE)
 
-set(ENABLE_OPENMP     "ON"        CACHE PATH "" FORCE)
-set(RAJA_ENABLE_OPENMP "ON"        CACHE PATH "" FORCE)
-
+set(ENABLE_OPENMP     "OFF"        CACHE PATH "" FORCE)
```

Comment:
No OpenMP?

Reply:
@sytuannguyen it's not working with `openmp`?

Reply:
NO! Below are some of the errors while I set it ON.

/workrd/SCR/GEOSX/l0359127/dev/GEOSX_dev/src/coreComponents/rajaInterface/GEOS_RAJA_Interface.hpp:34:34: error: ‘omp_parallel_for_exec’ in namespace ‘RAJA’ does not name a type
 using parallelHostPolicy = RAJA::omp_parallel_for_exec;
                                  ^~~~~~~~~~~~~~~~~~~~~
/workrd/SCR/GEOSX/l0359127/dev/GEOSX_dev/src/coreComponents/rajaInterface/GEOS_RAJA_Interface.hpp:35:34: error: ‘omp_reduce’ in namespace ‘RAJA’ does not name a type
 using parallelHostReduce = RAJA::omp_reduce;

Reply:
Ahh, you'd have to rebuild the tpls (or at least RAJA). This error to me looks like RAJA itself isn't built with OpenMP support. To do that just revert your changes to `ENABLE_OPENMP` and `RAJA_ENABLE_OPENMP`.

Reply:
Setting ON both ENABLE_OPENMP and RAJA_ENABLE_OPENMP leads to the error in the previous comment. 
What do you mean by rebuild tpls?

Reply:
That probably the TPLs (third party libraries) you are using are built without the `openmp` support for `RAJA`.
Do you use your own version or my version build from travis?

Reply:
We cannot rebuilt TPLs in Pangea because of a missing tool (related to lfs). We may need to rebuilt TPLs somewhere and copy to Pangea. 

Why openmp is needed if geosx can run without it?

Reply:
Using OpenMP significantly decreases the initialization time when each rank has a large partition (for example when using a GPU).

Reply:
Our CI indeed does not build for `openmp`. I was not really aware of that.
Shall we try to add support for this? that would be interesting.
I tried here https://github.com/GEOSX/GEOSX/pull/1261 and build fails: https://github.com/GEOSX/GEOSX/pull/1261/checks?check_run_id=1671724861

One time, a compilation error for centos/gcc/cuda/debug.
For ubuntu/gcc, each time `testKrylovSolvers` times out at `Hypre/KrylovSolverBlockTest/0.GMRES`. Strange.

Reply:
Let's integrate this like it is right now.
There are incoming actions for the `OpenMP` issue.